test/components/Auth/EnsureLoggedInTest.js

```diff
@@ -47,4 +49,78 @@ describe('EnsureLoggedIn', function test () {
     it('Should render children when no error is present and user is logged in', () => {
         expect(this.loggedInOutput.type).to.equal('div')
     })
+
+    context('propType passed in correctly', function () {
+        stubConsoleError()
+
+        let correctType = {
+            error: 'TESTTEST',
+            isLoggedIn: true,
+            store: GetMockStore({ auth: { error: 'Test Error' } })
+        }
+
+        describe('correct propTypes', function () {
+            it('should return false for no error with the propType', function () {
+                let result = setup(correctType, children)
+                let p = isPropTypeWarningMatch(console.error.args.toString())
+
+                sinon.assert.notCalled(console.error)
+                expect(p).to.be.false
+            })
+        })
+    })
+
+    context('propTypes passed in wrong', function() {
```

Comment:
As far as I can tell, these specs test the propTypes modules implementation. When testing, only test code you control. You aren't a propType maintainer, so don't test the behavior proTyprs provide.